### DISCUSSION TITLE
[1850570]死亡搁浅导演剪辑版 DEATH STRANDING DIRECTOR'S CUT

### DISCUSSION THREAD

**Discussion by zhuifengzhe1** [pjy612/SteamManifestCache#632] [Share]
### Steam 游戏ID

`1850570`

### 密钥列表

```hash
1850571;f0c67b235585f625ae33882c4547f9ef4404e96c5ad0a1fb4dbf43da61e04e24
1946740;e01735660cdb310784cc00ba796a0b48c48605b9b94cf3e4d473b3a5ae97af32
```

### 清单附件

[depotcache (2).zip](https://github.com/pjy612/SteamManifestCache/files/15152182/depotcache.2.zip)

### 备注

Code:39FD614C3C9F9E5A
Email:[asdadad331@outlook.com](mailto:asdadad331@outlook.com)
是个好游戏，该说不说修路给的赞是真的多

**Comment by pjy612** [comment]
* 密钥分隔符用错了

**Reply by zhuifengzhe1** [reply]
我改改
